docs/name-owners/atomic-name-trading/index.md

```diff
@@ -0,0 +1,294 @@
+---
+layout: page
+title: Atomic Name Trading
+---
+
+{::options parse_block_html="true" /}
+
+Because both currency transactions and name operations share a common technical backing (the Namecoin blockchain), it is possible to construct transactions that transfer both namecoins and a name _atomically_. This means that it is guaranteed that either both transfers succeed or both transfers are not executed at all. A potentially very useful application of this technique is trading of names for namecoins, since this makes it possible to sell or buy a name in a completely trustless manner without the necessity to use escrow.
+
+At the moment, it is possible to make such transfers only using RPC commands to namecoind directly. But in the future, hopefully an easy-to-use UI will be developed that handles all the technicalities. On this page, the basic work-flow will be described with RPC commands. We will assume as an example that Bob wants to buy 'd/my-cool-domain' from Sally for 10 NMC. The work-flow for this transaction is shown below on the test net. 
+
+## Bob Prepares a Buy-Offer
+
+Bob finds the domain of interest and Sally's offer to sell the domain. This could happen, for instance, via Sally using a special value for the domain or by some other channel out of the scope of this page. For instance: 
+
+```
+$ namecoind name_show d/my-cool-domain
```

Comment:
For all RPCs:  It should now be `namecoin-cli` instead of `namecoind`.

Reply:
Fixed.